fail2ban/tests/utils.py

```diff
@@ -123,7 +167,12 @@ def initProcess(opts):
 	#
 	stdout.setFormatter(Formatter(fmt))
 	logSys.addHandler(stdout)
-	#
+
+	# Let know the version
+	if opts.verbosity != 0: # pragma: no cover
```

Comment:
seriously -- why this code could/shouldn't be tested?  it could and should imho, thus 'no cover' is not warranted